sync/awsdatabricks.py

```diff
@@ -216,7 +221,7 @@ def _get_cluster_report(
 setattr(sync._databricks, "__claim", __name__)
 
 
-def _get_cluster_instances(cluster: dict) -> Response[dict]:
+def _get_cluster_instances(cluster: dict) -> Tuple[Response[dict], Response[dict]]:
```

Comment:
nit - Perhaps worth re-naming this function to something like `_get_cluster_aws_information`?